meta-refkit/recipes-image/images/refkit-installer-image.bb

```diff
@@ -23,10 +23,14 @@ REFKIT_INSTALLER_UEFI_COMBO () {
         rootfs=$4
         output_mounted=
         output_mountpoint=
+        output_luks=
+        LUKS_NAME=rootfs
+        LUKS_PASSWORD=refkit
```

Comment:
Ok, it wasn't clear to me from the commit that the password is not used by default. It just worries me a lot whenever I see a hardcoded password in the sources. :-) But are the `FIXED_PASSWORD` and `LUKS_PASSWORD` settings overlapping a bit, if one is used only if another is enabled? I mean, could we just say `FIXED_PASSWORD="refkit" image-installer`?